Level-2/trmv/trmv.cpp

```diff
@@ -29,14 +29,15 @@
 #include <stdlib.h>
 #include <vector>
 
+// lda and incx promoted to 64bit to avoid int32 overflow
 template <typename T>
 void referenceTrmvCalc(rocblas_fill    uplo,
                        std::vector<T>& A,
                        rocblas_int     N,
                        size_t          lda,
-                       std::vector<T>& workspace,
+                       std::vector<T>& x,
                        std::vector<T>& cpu_ref_result,
-                       rocblas_int     incx)
+                       ssize_t         incx)
```

Comment:
It seems that negative incx support is not provided in the reference function. Could we pass incx as std::abs(incx) and I could work on this later to add this support.

Reply:
> It seems that negative incx support is not provided in the reference function. Could we pass incx as std::abs(incx) and I could work on this later to add this support.

Well this is just an example of the function calls not meant to be trmv reference, so the negative inc support to the reference code can be added later.